Sources/Apollo/NetworkFetchInterceptor.swift

```diff
@@ -57,9 +60,15 @@ public class NetworkFetchInterceptor: ApolloInterceptor, Cancellable {
                            completion: completion)
       }
     }
+    
+    self.currentTask.mutate { $0 = task }
   }
   
   public func cancel() {
-    self.currentTask?.cancel()
+    guard let task = self.currentTask.value else {
```

Comment:
It seems this could just be `self.currentTask.value?.cancel()`, but I assume you prefer this because you find it more readable?

Reply:
somewhat, yep.